guides/common/modules/con_download-policies-overview.adoc

```diff
@@ -20,9 +20,23 @@ On Demand::
 This setting has no effect if you set a corresponding repository on a {SmartProxy} to *Immediate* because {ProjectServer} is forced to download all the packages.
 
 The *On Demand* policy acts as a _Lazy Synchronization_ feature because they save time synchronizing content.
+ifdef::satellite[]
 The lazy synchronization feature must be used only for `yum` repositories.
+endif::[]
+ifndef::satellite[]
+The lazy synchronization feature must be used only for `deb` and `yum` repositories.
+endif::[]
 You can add the packages to Content Views and promote to life cycle environments as normal.
 
+Note that some repositories such as EPEL not only add packages to their upstream repository, but also remove older versions of packages.
+You should enable *Mirror on Sync* and use download policy *immediate* for such repositories.
```

Comment:
`Mirror on Sync`- setting is replaced by `Mirroring Policy` in Katello-4.4 and latest -4.3, but I guess this will be a separate PR.

Reply:
Seconds too late to the party... But I can create a separate PR. We can track this in https://github.com/theforeman/foreman-documentation/issues/1184 for now.